README.md

```diff
@@ -19,33 +19,47 @@ Install the package with:
     or
     yarn add ordinalsbot
 
-## Usage
+## Import and intialization
 
 The package needs to be configured with your account's API key which you can get by opening a ticket in our Discord for now. Our developer dashboard is coming soon...
 
-``` js
-const ordinalsbot = require('ordinalsbot');
```

Comment:
@T1-O is there any way to use without import now or is it mandatory?
can someone use `require` ?